core/src/epicli/data/common/ansible/playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -179,7 +179,12 @@ setup.template.settings:
 # These settings control loading the sample dashboards to the Kibana index. Loading
 # the dashboards is disabled by default and can be enabled either by setting the
 # options here or by using the `setup` command.
-setup.dashboards.enabled: false
+setup.dashboards.enabled: true
```

Comment:
We cannot use `true` value here since Kibana is optional in Epiphany.

> When dashboard loading is enabled, Filebeat uses the Kibana API to load the sample dashboards. Dashboard loading is only attempted when Filebeat starts up. **If Kibana is not available at startup, Filebeat will stop with an error**.

My proposal is to make this property configurable (`false` / `true` / `auto`). `auto` would check if Kibana is available and set `false` if not.